src/utils/common.ts

```diff
@@ -359,3 +360,39 @@ export function detectProtocolByOwner(network: string, address?: string) {
   }
   return null;
 }
+
+export async function defaultAccountDiscovery(
+  isAccountUsed: Function,
+  getHdWalletAccountFromMnemonicSeed: Function,
+  protocol: string,
```

Comment:
If the protocol is not used in the logic but just returned at the end it makes no sense to use ti at all.